code/datums/brain_damage/magic.dm

```diff
@@ -33,17 +33,14 @@
 
 /datum/brain_trauma/magic/poltergeist/on_life()
 	..()
-	if(prob(4))
+	if(prob(10))
 		var/most_violent = -1 //So it can pick up items with 0 throwforce if there's nothing else
-		var/obj/item/throwing
 		for(var/obj/item/I in view(5, get_turf(owner)))
 			if(I.anchored)
 				continue
 			if(I.throwforce > most_violent)
 				most_violent = I.throwforce
-				throwing = I
-		if(throwing)
-			throwing.throw_at(owner, 8, 2)
+				I.throw_at(owner, 8, 2)
```

Comment:
don't do it this way, if you do, it'll throw more than a single item

Reply:
That's my point

Reply:
then make it truely random rather than a weirdly incrementing number

Reply:
It's random to get by multiple objects nearby?

Reply:
Making it more complex require more efforts

Reply:
no, it doesn't
this one would have it pick items in ascending force based on the order of the list returned by view()
if it immediately picks something with like 30 force, it'll only throw that
but if it somehow goes 1>2>3>4>....>29>30, then it would throw all 30 items
either make it a prob(#) or have it stop after it throws like 5 items or something

Reply:
Aigh